resources-manager/include/IResourceManager.hpp

```diff
@@ -15,13 +15,19 @@
 #include <cstdint>
 
 namespace concord::performance {
+
+struct PruneInfo {
+  uint64_t blocksPerSecond;
+  uint64_t batchSize;
+};
+
 class IResourceManager {
  public:
   virtual ~IResourceManager() = default;
   /*
-    getPruneBlocksPerSecond implementation computes proper pruning pace on its measuremnts.
+    getPruneBlocksPerSecond implementation computes proper pruning pace based on its measuremnts.
   */
-  virtual uint64_t getPruneBlocksPerSecond() const = 0;
+  virtual PruneInfo getPruneInfo() = 0;
```

Comment:
Is `IResourceManager` intended to be a generic interface or pruning-specific? If it is pruning-specific, above change is ok. Otherwise, I think it enforces it to be pruning-specific.

Reply:
Correct ResourceManger becomes pruningSpecific, so we thought of changin the name into sth like PruningStrategy

Reply:
Not sure, it should be done in this PR